compio-io/src/lib.rs

```diff
@@ -34,7 +34,7 @@
 //! ```
 //! use compio_buf::BufResult;
 //! use compio_io::AsyncRead;
-//! # #[compio_macros::main] async fn main() {
+//! # #[tokio::main(flavor = "current_thread")] async fn main() {
```

Comment:
If you want to show tokio, show it. Don't hide it. Make sure to point out that we don't necessarily need compio-runtime for io to work.